.github/workflows/ci.yml

```
@@ -27,6 +27,10 @@ jobs:
         include:
           - php: 7.4
             typo3: ^11.5
+          - php: 8.0
```

```suggestion
          - php: "8.0"
```

We need to ensure `8.0` is used as string here, otherwise Github normalizes this to just `8` which lets the Docker image lookup fail: https://github.com/pagemachine/typo3-phinx/runs/4948152273#step:3:9 (see https://github.com/thecodingmachine/docker-images-php#images)